pkg/interface/src/views/App.js

```diff
@@ -108,7 +108,11 @@ class App extends React.Component {
   faviconString() {
     let background = '#ffffff';
     if (this.state.contacts.hasOwnProperty('/~/default')) {
-      background = `#${uxToHex(this.state.contacts['/~/default'][window.ship].color)}`;
+        if (this.state.contacts['/~/default'].hasOwnProperty(window.ship)) {
```

Comment:
Wouldn't the following be cleaner?

```
background = this.state.contacts?.['/~/default']?.[window.ship]?.color ? `#${uxToHex(this.state.contacts['/~/default'][window.ship].color)}` : '#ffffff';`
```

Also format the commit `interface: commit name`.

Reply:
It would, I was just matching the existing check. Will update.